mithril-common/src/entities/cardano_database.rs

```diff
@@ -80,6 +80,9 @@ pub enum DigestLocation {
         /// URI of the cloud storage location.
         uri: String,
     },
+    /// Other location not known.
```

Comment:
Maybe this is a bit more clear (here and in the other locations as well):
```suggestion
    /// Catchall for unknown location variants.
```

Reply:
Done